src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/IframeRunner.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.function.Supplier;
+
+public class IframeRunner {
+  private final WikiaWebDriver driver;
+  private final Wait wait;
+
+  IframeRunner(WikiaWebDriver driver) {
+    this.driver = driver;
+    this.wait = new Wait(driver);
+  }
+
+  public <T> T usingIframeGet(By iframeSelector, final Supplier<T> f) {
+    WebElement iframe = wait.forElementPresent(iframeSelector);
+    return usingIframeGet(iframe, f);
+  }
+
+  private <T> T usingIframeGet(WebElement iframe, final Supplier<T> f) {
+    driver.switchTo().frame(iframe);
+    T result = f.get();
+    driver.switchTo().defaultContent();
+    return result;
+  }
+
+  public void usingIframe(By iframeSelector, Runnable f) {
+    WebElement adIframe = wait.forElementPresent(iframeSelector);
+    usingIframe(adIframe, f);
+  }
+
+  private void usingIframe(WebElement iframe, Runnable f) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "usingIframe" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
It's used in line 33 :|